### DISCUSSION TITLE
Cheap camera suggestions

### DISCUSSION THREAD

**Discussion by MisterWho** [openbookings26/ContaCam#24] [Q&A] [answered]
Hello everyone.

I've been shopping for replacement cameras, and I'm not too demanding when it comes to features or build quality - but something that's been p--sing me off is the fact that every cheap camera I buy demands me to install a smartphone app and share my location (why???) when I just want to plugin a lan cable and give it an IP address. Also it seems that web based configuration is being discarded in favour of these crappy apps.
Anyone has any suggestions of cheap-ish cameras with no strings attached that I could buy?
Best regards!

**Comment by openbookings26** [comment, marked as answer]
I use Foscam R4S. They have LAN and WiFi, PTZ and SD card for local storage. Easy to configure and you don't have to use their cloud features if you don't want to. You can find them on amazon for aroun $50-$60.

**Comment by WouterJD** [comment]
Hi @MisterWho what I did is the following:

**Camera**
- IP camera from Ali express
- Install using the smartphone app

**Network**
- Go to the network console of the wifi router and assign a fixed address to the camera; see [21-05-contacam-networking/89-network-ip-camera-setup-and-troubleshoot](https://www.contaware.com/help/manual-faqs/21-05-contacam-networking/89-network-ip-camera-setup-and-troubleshoot.html)
- Now the IP-address of the camera is known and fixed
- Disallow internet access to that IP-address (there's no need the world has access to my camera; only contacam needs access)

**Contacam**
- Use the fixed IP-address in contacam, which must be running in a subnet where the camera is connected to have connectivity

**Smartphone app**
- The smartphone app remains for configuration purpose only

**Comment by MisterWho** [comment]
Thank you all for your replies.

In the meantime, since I found out I could buy Uniarch 1080p cameras with a full-featured web browser page for configuration and no need for apps, and for less money than I pay for chinese cameras, I never went back.

https://www.orbitadigital.com/pt/cctv-ip/cameras-ip/bullet/54327-uniarch-uv-ipc-b122-apf28-eco-camara-bullet-ip-uniview-uniarch-2-mp-lente-28.html

**Comment by WouterJD** [comment]
Configuting through a webpage on the camera is the best approach; no local software needed (other than contacam)
